ses/data.tf

```diff
@@ -0,0 +1,3 @@
+data "aws_ssm_parameter" "notification_email_prefix" {
+  name = "/mgmt/notification/email/prefix"
```

Comment:
Out of curiosity, what's the reason for moving the prefix into the parameter store rather than the whole email address? I'd have thought it would be simpler to add the whole address, and it's more obvious what it means when you see it in the parameter store too.

Reply:
If you look in the changes for the `ses/main` file, the email is split into `prefix/domain` which makes the ses module more generic as it can cope with non nationalarchives domains. I didn't want to make the ses module less generic and I didn't want to have two parameters, one for the full email and one for the prefix so I settled on the prefix.

Reply:
Sorry, still a bit confused. Why would you need two parameters? Wouldn't it work with just a single parameter that takes a whole email address?

This isn't a big deal. 🙂 I just thought it would be easier to follow the code if we store whole email addresses rather than storing the prefix separately to the domain.

Reply:
The ses email is set as
```
resource "aws_ses_email_identity" "email_address" {
  email = "${local.email}@${var.domain}"
```
If the `local.email` parameter comes from the parameter store then it would be `prefix@nationalarchives.gov.uk@${var.domain}`

Reply:
Yes, so why not skip the `${var.domain}` bit in the SES config? Sorry if I'm just missing something really obvious! 🙂 

```
resource "aws_ses_email_identity" "email_address" {
  email = "${local.email}"
```

Reply:
You can, it would work but this keeps the domain explicit, as it is in the rest of the ses module. If I do the whole email address, it means you can have an email with a different domain as the rest of the SES setup like this:
```
module "ses" {
  source = "./tdr-terraform-modules/ses"
  domain = "somethingelse"
  email_address = "test@nationalarchives.gov.uk"
}
```
which probably isn't the end of the world but it is changing the behaviour of the module which is what I was trying to avoid.

Reply:
We had a chat about this earlier. I think it would make more sense to have the email address as a single parameter, because it's kind of coincidental that the alert email domain is the same as the web domain, but it's not a huge issue.

We'll revisit it if they ever need to be separated, or if it causes some huge confusion when we try to reuse the module elsewhere.